test/membrane_rtc_engine/engine_test.exs

```diff
@@ -101,8 +102,11 @@ defmodule Membrane.RTC.EngineTest do
       )
 
       assert_receive {:track_metadata_updated, %Track{id: ^track_id, metadata: "new-metadata"}}
+
+      :ok = Engine.terminate(rtc_engine)
```

Comment:
Let's put this in `on_exit` as `assert :ok == Engine.terminate(rtc_engine, blocking?: true)`